Objects/genericaliasobject.c

```diff
@@ -651,7 +651,7 @@ Py_GenericAlias(PyObject *origin, PyObject *args)
         return NULL;
     }
     if (!setup_ga(alias, origin, args)) {
-        PyObject_GC_Del((PyObject *)alias);
+        Py_DECREF(alias);
```

Comment:
This is going to fail because the object is not tracked. Try for yourself by allowing this path to be taken.

Check out the diff in my last PR

Reply:
I can't believe I didn't notice this again :(. Thanks.

Reply:
Just did a check of `ga_new` to make sure and it seems we don't need to track there (I think). `type->tp_alloc` uses `PyType_GenericAlloc` which auto tracks if `Py_TPFLAGS_HAVE_GC` is set.

Reply:
That is correct, I think @vstinner wants to introduce some new API to not track when calling that function, but that is a story for another day

Reply:
I only would like to start tracking newly created objects later, but not change deallocator functions.

Reply:
https://bugs.python.org/issue44531